.github/workflows/maven.yml

```diff
@@ -25,4 +25,7 @@ jobs:
         cache: maven
     - name: Build with Maven
       run: mvn -B package --file pom.xml
-      
```

Comment:
Maybe I could do this on the `maven-publish.yml` workflow and simply increment by 1. WDYT?

Reply:
@pravinbhat what do you think?